app/Console/Kernel.php

```diff
@@ -24,14 +24,14 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        $schedule->command('db:refresh') // use custom command to make sure that the commands are chained
+        $schedule->command('db:refresh --backup') // use custom command to make sure that the commands are chained
             ->daily() // Run daily at midnight
             ->environments(['staging', 'dev', 'local']) // only run for APP_ENV tagged staging, dev, or local
```

Comment:
I guess after we do a real production release, we'll change this to run the backup on Production so that we can replicate the data to staging. Is that correct?

Reply:
Yeah. I expected it would run on both. 
We can have a shared directory between both production and sandbox.
I can add in a command option to allow to pull from different locations?
Example:
Backups occur to `/data/[environment]/seeds` by default. 
If you add `--path=/data/production/seeds` while in sandbox it would restore from production seeds.

Reply:
I think this might be a better solution.
We can just tag the files with an extension that is the environment.
So for example backing up on my local which has **local** set at the environment then the backup files look like this.
```
identities.json.local  interpretations.json.local  resource_collections.json.local  resources.json.local  topics.json.local
```
So by adding `--env=staging` to any of the commands it will change the relevant files that will be handled.

What do you think of this solution?

Reply:
If possible can we move the "tag" between the file name and the ".json" extension. So something like "identities.local.json". That way applications (e.g. an IDE) would still know the file is a JSON file.